ndc-reference/bin/reference/main.rs

```diff
@@ -13,9 +16,12 @@ use axum::{
 };
 
 use indexmap::IndexMap;
-use ndc_client::models::{self, LeafCapability, RelationshipCapabilities};
+use ndc_client::models::{
+    self, LeafCapability, NestedArray, NestedField, NestedObject, RelationshipCapabilities,
```

Comment:
Consider referencing these qualified as `models::NestedArray` etc. just to keep the imports under control here.